### PR TITLE
Fixes and tweaks based on Alpha Testing feedback

### PR DESCRIPTION
**Pull request by hcarneiro**
The commits should explain the changes.
You can also have a look at the [spreadsheet with the "Fixed" filter view applied here](https://docs.google.com/spreadsheets/d/1XcSpuLsa-dxdcE9hF0OOfBi8JzzTE-PGDq8qUcX-PtM/edit#gid=0&fvid=1489352042).

### CODE REVIEW

File: js/interface/add-markers.js [hunk 1]
```diff
@@ -70,7 +70,8 @@ Fliplet.InteractiveMap.component('add-markers', {
       },
       selectedPinchMarker: undefined,
       tappedMarkerId: undefined,
-      saveDebounced: _.debounce(this.saveToDataSource, 1000)
+      saveDebounced: _.debounce(this.saveToDataSource, 1000),
+      dsErrors: false
```
Comment (squallstar) [inline js/interface/add-markers.js]: dsErrors?

File: js/interface/add-markers.js [hunk 2]
```diff
@@ -100,14 +101,14 @@ Fliplet.InteractiveMap.component('add-markers', {
...
-            size: markerData ? markerData.size : '24px',
-            positionx: marker.data[this.markerXPositionColumn],
-            positiony: marker.data[this.markerYPositionColumn],
+            name: marker.data[this.markerNameColumn] || 'Data source marker',
+            map: marker.data[this.markerMapColumn] || this.widgetData.maps[0].name,
+            type: marker.data[this.markerTypeColumn] || this.widgetData.markers[0].name,
+            icon: markerData ? markerData.icon : this.widgetData.markers[0].icon,
+            color: markerData ? markerData.color : this.widgetData.markers[0].color,
+            size: markerData ? markerData.size : this.widgetData.markers[0].size,
+            positionx: marker.data[this.markerXPositionColumn] || '100',
```
Comment (squallstar) [inline js/interface/add-markers.js]: positionX and positionY (use the capitals)

File: js/libs/interface.js [hunk 3]
```diff
@@ -178,6 +199,7 @@ const app = new Vue({
         .then(() => {
           if (!stopComplete) {
             Fliplet.Widget.complete()
+            Fliplet.Studio.emit('reload-widget-instance', widgetId);
```
Comment (squallstar) [inline js/libs/interface.js]: unnecessary `;`

File: js/interface/add-markers.js (hunk 1 above)
Comment (hcarneiro) [reply js/interface/add-markers.js]: Updated to `dsConfigError`

Comment (squallstar) [reply js/interface/add-markers.js]: @hcarneiro what's `ds` for? dataSource?

**Pull request merged**